src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
I guess the code should still work if you get rid of these values (try leaving the keys only)
```suggestion
    name,
    wheels,
    version,
```

Reply:
yea its work, i fogot of this thanks